### PR TITLE
Add Sprig first game workshop

### PR DESCRIPTION
**Pull request by jianmin-chen**
This is based off [this](https://github.com/hackclub/sprig/blob/main/docs/GETTING_STARTED.md), with a couple of tweaks to make it more workshop-friendly (hopefully).

### CODE REVIEW

File: workshops/sprig_first_game/README.md [hunk 1]
```diff
@@ -0,0 +1,479 @@
...
+................
+.......0........
+.....00.000.....
+....0.....00....
+....0.0.0..0....
+....0......0....
+....0......0....
+....00....0.....
+......00000.....
+......0...0.....
```
Comment (sampoder) [inline workshops/sprig_first_game/README.md:63]: Could we give this section a bit more context? The code block here is massive :D

File: workshops/sprig_first_game/README.md [hunk 2]
```diff
@@ -0,0 +1,479 @@
...
+    }
+  }
+});
+```
+
+Sprig makes it easy to write fun games!
+
+## Uploading to the Gallery
+
+So you've written a game and want to share it. Learn how to share your games [here](https://sprig.hackclub.com/share)! Have fun!
```
Comment (sampoder) [inline workshops/sprig_first_game/README.md:478]: Could we expand this section a little and have it earlier in the workshop?

**Review by sampoder — CHANGES_REQUESTED** [review]
Great workshop! Two quick comments from me:

File: workshops/sprig_first_game/README.md (hunk 1 above)
Comment (jianmin-chen) [reply workshops/sprig_first_game/README.md:63]: No problem! I'll write up an explanation for it.

File: workshops/sprig_first_game/README.md (hunk 2 above)
Comment (jianmin-chen) [reply workshops/sprig_first_game/README.md:478]: Yep, will do!

**Comment by jianmin-chen** [comment]
Ok @sampoder, I've added a couple of tweaks - take a look at them now!

File: workshops/sprig_first_game/README.md [hunk 3]
```diff
@@ -0,0 +1,540 @@
...
+        if (currentLevel !== undefined) {
+            setMap(currentLevel);
+        } else {
+            addText("you win!", { y: 4, color: color`3` });
+        }
+    }
+});
+```
+
+Sprig makes it easy to write fun games!
```
Comment (sampoder) [inline workshops/sprig_first_game/README.md:540]:
```suggestion
```

**Review by sampoder — APPROVED** [review]
LGTM! Thank you @jianmin-chen!

**Pull request merged**